setup.py

```diff
@@ -14,5 +14,6 @@
     install_requires=[
         "pythunder",
         "pycyclone",
+        "pillow",
```

Comment:
This is not a hard requirement. Can you make it optional so people can install it as `archipelago[vis]` or something similar?